charts/mailhog/values.yaml

```diff
@@ -108,13 +119,13 @@ extraEnv: []
 # ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes
 livenessProbe:
   tcpSocket:
-    port: 1025
+    port: *containerSmtpPort
```

Comment:
I see, what you try to achieve here - but imho this is not considered best practice to use the yaml anchors with helm.

i'd say, tackle the behaviour you want to achieve in the templates directly - so skip the definition in the _defaults_ here and access containerPort in the template or livenessProbe Port when defined.